pkg/vz/vm_darwin.go

```diff
@@ -353,12 +353,12 @@ func attachNetwork(driver *driver.BaseDriver, vmConfig *vz.VirtualMachineConfigu
 				if runtime.GOOS != "darwin" {
 					return fmt.Errorf("networks.yaml '%s' configuration is only supported on macOS right now", nw.Lima)
 				}
-				socketVMNetOk, err := nwCfg.IsDaemonInstalled(networks.SocketVMNet)
+				socketVMNetOk, err := config.IsDaemonInstalled(networks.SocketVMNet)
```

Comment:
nwCfg isn't fine?

Reply:
Yeah, I was unsure about it, but then decided to change it for consistency. Any time `networks.LoadConfig()` is called, the return value is stored in `config`. This made more sense to me than using different names in different places.

But I agree it is confusing inside `qemu`, which has its own `Config` type. How about calling it `config` inside the `networks` package and `nwConfig` everywhere else?

I'm also fine with using `cfg` and `nwCfg`, but again would want to use these names then consistently everywhere.

Reply:
nwCfg is more descriptive than config

Reply:
Ok, new attempt:

A `network.Config` variable is called:

* `c` when it is the receiver
* `cfg` inside the `networks` package
* `nwCfg` elsewhere

I adjusted some other "config" names in the qemu package, but not in vz, as that would modify over 100 lines of unrelated configs, which I will deal with once we agree on a naming scheme.

@AkihiroSuda PTAL